### PR TITLE
Link to /TR

### PR DESCRIPTION
**Pull request by xfq**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/picture-in-picture/pull/176.html" title="Last updated on Feb 4, 2020, 12:59 AM UTC (e30c229)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/picture-in-picture/176/9a28dfa...e30c229.html" title="Last updated on Feb 4, 2020, 12:59 AM UTC (e30c229)">Diff</a>

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@tidoust Can you confirm this PR is valid?

File: index.bs [hunk 1]
```diff
@@ -4,6 +4,8 @@ Shortname: picture-in-picture
 Level: 1
 Status: ED
 ED: https://w3c.github.io/picture-in-picture/
+TR: https://www.w3.org/TR/picture-in-picture/
+Previous Version: https://www.w3.org/TR/2020/WD-picture-in-picture-20200130/
```
Comment (tidoust) [inline index.bs]: I wouldn't necessarily include a link to the previous published Working Draft in the Editor's Draft (that's the sort of thing that are tedious to maintain and may easily become wrong)

Comment (xfq) [reply index.bs]: I've just removed this line.

**Comment by beaufortfrancois** [comment]
I've merged as feedback has been addressed. Thank you @xfq!

**Pull request merged**